.github/workflows/sphinx-doc.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow will generate the Sphinx documentation
+
+name: Sphinx Documentation
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+    # paths: [ "documentation/**"]
```

Comment:
i assume you meant to only generate the documentation only when doc files are changed? And you had commented this out to test the pipelines?

Reply:
Thanks for the review. You were actually too early. I was just testing the action.
There seems to be no easy way to test these things, you just need to crate the action in such a way that it runs for your Pull Requests and when when ready set-up the triggers properly.